src/main/java/games/strategy/triplea/image/ImageRef.java

```diff
@@ -15,16 +15,16 @@
  * getImage method ensures that the image will be loaded before returning.
  */
 class ImageRef {
-  public static final ReferenceQueue<Image> s_referenceQueue = new ReferenceQueue<>();
-  public static final Logger s_logger = Logger.getLogger(ImageRef.class.getName());
-  private static final AtomicInteger s_imageCount = new AtomicInteger();
+  private static final ReferenceQueue<Image> referenceQueue = new ReferenceQueue<>();
+  private static final Logger logger = Logger.getLogger(ImageRef.class.getName());
```

Comment:
These fields are not used outside this class, so I reduced their visibility at the same time I renamed them.